src/components/database/DatabaseOverview.vue

```diff
@@ -0,0 +1,250 @@
+            :name="databaseToManipulate"
+            :filter="customFilterStore.getFilter(databaseToManipulate)"
+            @edit="confirmEditDatabase"
+        />
+    </div>
+</template>
+
+<script setup lang="ts">
+import {ref, computed, onMounted} from 'vue';
+import useCustomFilterStore, {Filter, FilterType} from "@/store/new/CustomFilterStore.js";
+import TaxonomyResponseCommunicator from "@/logic/communicators/unipept/taxonomic/TaxonomyResponseCommunicator";
+import {DEFAULT_API_BASE_URL, DEFAULT_ONTOLOGY_BATCH_SIZE} from "@/logic/Constants";
+import ProteinResponseCommunicator from "@/logic/communicators/unipept/protein/ProteinResponseCommunicator";
+import UniprotCommunicator from "@/logic/communicators/uniprot/UniprotCommunicator";
+import {useNumberFormatter} from "@/composables/useNumberFormatter";
+import useProteinOntology from "@/composables/ontology/useProteinOntology";
+import useProteomeOntology from "@/composables/ontology/useProteomeOntology";
+import CreateCustomDatabase from "@/components/database/CreateCustomDatabase.vue";
+import DeleteDatabaseDialog from "@/components/database/DeleteDatabaseDialog.vue";
+import EditCustomDatabase from "@/components/database/EditCustomDatabase.vue";
+import EditDatabaseDialog from "@/components/database/EditDatabaseDialog.vue";
+
+const { ontology: proteinOntology, update: updateProteinOntology } = useProteinOntology();
+const { ontology: proteomeOntology, update: updateProteomeOntology } = useProteomeOntology();
+const customFilterStore = useCustomFilterStore();
+const { formatNumber } = useNumberFormatter();
+
+const taxonomyCommunicator = new TaxonomyResponseCommunicator(DEFAULT_API_BASE_URL, DEFAULT_ONTOLOGY_BATCH_SIZE);
+const proteinCommunicator = new ProteinResponseCommunicator(DEFAULT_API_BASE_URL, DEFAULT_ONTOLOGY_BATCH_SIZE);
+
+const emits = defineEmits<{
+    (e: 'database:update', name: string, newName: string, newFilter: Filter): void,
+    (e: 'database:delete', name: string): void,
+}>();
+
+const databaseDialogOpen = ref(false);
+const deleteDatabaseDialogOpen = ref(false);
+const editDatabaseDialogOpen = ref(false);
+const databaseToManipulate = ref<string>('');
+const taxonCounts = ref<Map<string, string>>(new Map());
+const proteinCounts = ref<Map<string, string>>(new Map());
+
+const databases = computed(() => {
+    return customFilterStore.filters.map(name => {
+        const filter = customFilterStore.getFilter(name);
+        return {
+            name: name,
+            type: showFilterType(filter.filter),
+            taxaCount: taxonCounts.value.get(name) || 'computing...',
+            proteinCount: proteinCounts.value.get(name) || 'computing...',
+        };
+    });
+});
+
+const showFilterType = (filterType: FilterType) => {
+    switch (filterType) {
+        case FilterType.UniProtKB:
+            return 'N/A';
+        case FilterType.Taxon:
+            return 'Taxa';
+        case FilterType.Protein:
+            return 'Proteins';
```

Comment:
Maybe change this description to `Protein Accession Numbers`? Because the database always contains proteins, this might be a bit confusing for the users.